core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -76,17 +76,17 @@ THE SOFTWARE.
   </st:documentation>
   <d:taglib uri="local">
     <d:tag name="body">
-      <table style="width:100%">
+      <div style="width:100%">
         <j:set var="help" value="${descriptor.helpFile}" />
         <j:if test="${hasHeader}">
-          <tr>
-            <td colspan="3">
+          <div class="tr help-sibling">
```

Comment:
could you change `tr` by a meaningful name? tr denotes table-related stuff and we are trying to get rid of them.

Reply:
The class is named like that to make it easier to provide backwards compatibility. For example, it makes easier to migrate JS selectors.

Reply:
I don't agree on the approach because we will have a debt here for the sake of the migration, but I understand it.